tgui/packages/tgui-panel/chat/constants.js

```diff
@@ -5,11 +5,11 @@
  */
 
 export const MAX_VISIBLE_MESSAGES = 2500;
-export const MAX_PERSISTED_MESSAGES = 1000;
+export const MAX_PERSISTED_MESSAGES = 10000;
 export const MESSAGE_SAVE_INTERVAL = 10000;
 export const MESSAGE_PRUNE_INTERVAL = 60000;
 export const COMBINE_MAX_MESSAGES = 5;
-export const COMBINE_MAX_TIME_WINDOW = 5000;
+export const COMBINE_MAX_TIME_WINDOW = 50000;
```

Comment:
> We use MS rather than DS for seconds in js.

Might be good to leave a comment here considering the confusion arose once already?

Reply:
Put a comment there

Reply:
50 seconds? That's too much, old examine messages will stack if not changed